PR-Trigger/index.ts

```diff
@@ -37,34 +34,31 @@ const httpTrigger = async function (context, _req) {
 
     // Allow the bot to run side-effects which are not the 'core' function
     // of the review cycle, but are related to keeping DT running smoothly
-    const sideEffects = {
-        "check_suite": mergeCodeOwnersOnGreen
+    if (event === "check_suite") {
+        await mergeCodeOwnersOnGreen(req.body as EventPayloads.WebhookPayloadCheckSuite);
     }
 
-    if (sideEffects[event]) sideEffects[event](req.body);
-
     // https://developer.github.com/webhooks/
-    const acceptedEventsToActions = {
+    const acceptedEventsToActions: Record<string, string[] | undefined> = {

```

Comment:
Heh, you can always turn on http://typescriptlang.org/tsconfig#noUncheckedIndexedAccess